### ISSUE TITLE
Missing data in birdsnap/test

### ISSUE THREAD

**Issue by KarolinaPondel** [ml-jku/cloob#17]
I tried to run **src/notebooks/zeroshot.ipynb**, but but unfortunately after loading the data (dataloader) there is a 
'''
FileNotFoundError: Found no valid file for the classes 046, 066, 123, 299, 302, 351, 403, 436, 465. Supported extensions are: .jpg, .jpeg, .png, .ppm, .bmp, .pgm, .tif, .tiff, .webp. 
'''
However, it is possible to run the rest of the code despite this and I get a pretty good result, strange.... In the second scenario I removed the missing folders, everything went through without error, however in the results the bird images shown have mixed up captions....

**Comment by fuersta** [comment]
Yes, unfortunately there are some classes of birdsnap missing, which are not available to download anymore.
The torchvision package we used during development did allow empty classes, with the current version you have to set a flag in the dataset called `allow_empty`. 
https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html
If you remove the empty folders, you get wrong class allocation leading to wrong results!
